pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling", "hatch-vcs>=0.3", "setuptools-scm>=7.1", "hatch-requirements-txt"]
+
+[project]
+author = "Anaconda, Inc."
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Console",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+]
+description = "Client library and CLI plugin for anaconda.org"
+dynamic = ["dependencies", "optional-dependencies", "version"]
+license = {text = "BSD-3-Clause"}
+name = "anaconda-client"
+readme = "README.md"
+requires-python = ">=3.8"
+urls = {source = "https://github.com/anaconda/anaconda-client"}
+
+[project.entry-points."anaconda_cli.subcommand"]
+org = "binstar_client.plugins:app"
+
+[project.scripts]
+# Legacy entrypoints
+binstar = "binstar_client.scripts.cli:main"
+conda-server = "binstar_client.scripts.cli:main"
+
+[tool.distutils.bdist_wheel]
+universal = true
+
+[tool.hatch.build.hooks.vcs]
+template = """
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+# pylint: disable=inconsistent-quotes
+# pylint: disable=missing-module-docstring
+__version__ = {version!r}
+"""
+version-file = "binstar_client/_version.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
```

Comment:
Would you maybe want to include the license, too?

Reply:
Good call. I grabbed this from elsewhere but I'm actually not sure how this differs from the default. I'll check.

Reply:
Done